examples/python/hackathon_app/README.md

```diff
@@ -87,6 +87,27 @@ yarn start
 
 - this launches the frontend static server at http://127.0.0.1:3000 which proxies fetch requests to the backend server
 
+## Bulk import script
+
+The `bulk_import.py` script reads a CSV file that has "first_name", "last_name", and "email" column headers and adds users to the hack instance.
+It can be re-run safely w/out duplicating users (primary key is "email"). It will send the user an account setup email.
```

Comment:
```suggestion
It can be re-run safely w/out duplicating users (primary key is "email"). It will send newly registered users an account setup email.
```